### PR TITLE
EXLM-1206: Browse Filters: Ability to change the font size of the topics header

### PR DESCRIPTION
**Pull request by gandal-adobe**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: EXLM-1206

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/
- After: https://feature-exlm-1206--exlm--adobe-experience-league.hlx.live/

### CODE REVIEW

File: blocks/browse-filters/browse-topics.css [hunk 1]
```diff
@@ -54,11 +54,35 @@
...
+        font-size: var(--spectrum-font-size-300);
+    }
+
+    .browse-topics-item.h5 {
+        font-size: var(--spectrum-font-size-200);
+    }
+
+    .browse-topics-item.h6 {
+        font-size: var(--spectrum-font-size-100);
+    }
```
Comment (noormd08) [inline blocks/browse-filters/browse-topics.css:80]: Any reason for adding these font sizes specific to Desktop? We generally don't override the header fonts in the block level unless there is specific need for it.

File: blocks/browse-filters/browse-filters.js [hunk 2]
```diff
@@ -1108,7 +1109,7 @@ function decorateBrowseTopics(block) {
       .forEach((topicsButtonTitle) => {
         const parts = topicsButtonTitle.split('/');
         const topicName = parts[parts.length - 1];
-        const topicsButtonDiv = createTag('button', { class: 'browse-topics browse-topics-item' });
+        const topicsButtonDiv = createTag('button', { class: `browse-topics browse-topics-item ${tagFontStyle}` });
```
Comment (noormd08) [inline blocks/browse-filters/browse-filters.js:1112]: I hope the ask is to update the heading for the topics but the code changes made on each topic (button).

Comment (gandal-adobe) [reply blocks/browse-filters/browse-filters.js:1112]: @noormd08 - i misread that. It is the header and not the tag buttons. thanks!

**Comment by gandal-adobe** [comment]
Closing this PR and creating a new branch as there were changes to affected files and attempt to merge got messy.

**Pull request closed without merging**